src/components/organisms/about/about.scss

```
@@ -0,0 +1,23 @@
+@use "../../variables" as *;
+
+.images {
+  width: 100%;
```

```suggestion
.organisms-about {
  .images {
    width: 100%;
```
現状スコープ（影響範囲）が広すぎるので、今回作る about コンポーネント内の外に影響を及ぼさないようにしましょう。
このようにHTMLの構造通りの入れ子構造をscssの方にも記述するのがおすすめです。

---

対応しました!

0576e250a28af08f4a9b077ae50e425ac3420219